### PR TITLE
Needs deletion

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: temba/archives/migrations/0005_auto_20180601_1836.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("archives", "0004_auto_20180522_1502")]
+
+    operations = [
+        migrations.RemoveField(model_name="archive", name="is_purged"),
+        migrations.AddField(
```
Comment (rowanseymour) [inline temba/archives/migrations/0005_auto_20180601_1836.py]: is this supposed to be an Remove,Add rather than a Rename?

Comment (rowanseymour) [reply temba/archives/migrations/0005_auto_20180601_1836.py]: now I can't remember if is_purged means it has been deleted or or it needs deleted... +1 for the rename

Comment (nicpottier) [reply temba/archives/migrations/0005_auto_20180601_1836.py]: Feels weird to do a rename when the semantic meaning is opposite.

Comment (rowanseymour) [reply temba/archives/migrations/0005_auto_20180601_1836.py]: right I just forgot what that field means

**Pull request merged**